test/middleware/spec.back-link.js

```diff
@@ -168,6 +175,7 @@ describe('Back Links', function () {
 
     it('returns undefined if referrer header is not on whitelist', function () {
         req.get.withArgs('referrer').returns('http://example.com/not-whitelisted');
+        req.baseUrl = '/not-whitelisted';
```

Comment:
I have explicitly assigned the correct value to `req.baseUrl` in all cases to illustrate that the stub with 'referrer' will not create a baseUrl and to avoid assumptions about tests.